docs/en/engines/table-engines/mergetree-family/replication.md

```diff
@@ -113,6 +113,8 @@ You can have any number of replicas of the same data. Yandex.Metrica uses double
 
 The system monitors data synchronicity on replicas and is able to recover after a failure. Failover is automatic (for small differences in data) or semi-automatic (when data differs too much, which may indicate a configuration error).
 
+You can use a special logic for performing merges (for more information, see the documentation for [execute_merges_on_single_replica_time_threshold](../../../operations/settings/settings.md#execute-merges-on-single-replica-time-threshold) setting). 
```

Comment:
Not sure if we should advertise it too much :) It has sense only in some limited use cases when merges are CPU bounded not IO bounded.